src/scripts/records/AsyncTaskStatusRecord.js

```diff
@@ -1,16 +1,23 @@
 /* @flow */
```

Comment:
Now these record constructors take in plain JS objects and deeply convert child properties to immutable structures as needed